### DISCUSSION TITLE
Tracking down warnings

### DISCUSSION THREAD

**Discussion by danatstudy** [mydumper/mydumper#1678] [Q&A] [answered]
Hey everyone! On the round of dumping and loading I've been doing recently, I noticed a lot of the following types of errors in the logging. 

```
2024-11-22 18:01:37 [WARNING] - Connection 2553: Warnings found during INSERT between lines: 6116117 and 6123742: (null)
2024-11-22 18:01:41 [WARNING] - Connection 2553: Warnings found during INSERT between lines: 6123743 and 6131373: (null)
2024-11-22 18:01:46 [WARNING] - Connection 2553: Warnings found during INSERT between lines: 6131374 and 6139014: (null)
``` 

I'm up to about a thousand of those right now on a very long running loading session. Does anyone know how to track down just which table and part that these are coming from? There are log entries all around them with the various threads, but nothing identifiable. 

Also, does anyone know if this means I am missing data between those rows?

I'm using mydumper/myloader v0.16.9-1 for both the dump and load. The source server was CentOS 7 and new server is Rocky 9, both running Percona Server 5.7. 

myloader started with:
```
myloader -u root -p `cat pass` -B ${db} -t 60 -v 3 -d /r1/dumps_orig/${db} --fifodir=/tmp/bkp --serialized-table-creation --max-threads-per-table 4 --max-threads-for-index-creation 4 -L ${db}.myloader.log
``` 

Thanks!

**Comment by davidducos** [comment, marked as answer]
Hi @danatstudy I've updated the documentation https://mydumper.github.io/mydumper/docs/html/myloader_usage.html#cmdoption-myloader-show-warnings.

**Reply by danatstudy** [reply]
Thanks much for that @davidducos!
